sdk/compiler/damlc/tests/src/DA/Test/DamlcUpgrades.hs

```diff
@@ -874,6 +619,33 @@ tests damlc =
             createDirectoryIfMissing True (takeDirectory $ dir </> file)
             writeFileUTF8 (dir </> file) content
 
+data TestOptions = TestOptions
```

Comment:
This makes things much more readable! Thanks.

Reply:
:)